### ISSUE TITLE
Issue with transformer guide

### ISSUE THREAD

**Issue by jasonzhang2022** [tensorflow/tensorflow#28613]
## URL(s) with the issue:
https://www.tensorflow.org/alpha/tutorials/text/transformer
## Description of issue (what needs changing):
format and normalization layer.

### Clear description
1. when describe the multi-head attention, there is this text in one line:  "Multi-head attention consists of four parts: * Linear layers and split into heads. * Scaled dot-product attention. * Concatenation of heads. * Final linear layer. "
  I guess the * mean bullet items. It is not formatted correctly.
2. The EncoderLayer and DecoderLayer use LayerNormalization. There is no LayerNormalization in keras.layers. There is only BatchNormalization.
3. Evaluate step creates mask. No mask is needed since a) there is no padding for a single sentence. b) there is no look_ahead since we are trying to predict next words.

**Comment by jvishnuvardhan** [comment]
@jasonzhang2022 Thanks for finding this. First one  was already updated in `master` branch [here](https://github.com/tensorflow/docs/blob/master/site/en/r2/tutorials/text/transformer.ipynb) and it will be correctly rendered when the update are pushed to website. We will take a look at the rest of the two issues. Thanks!

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by jvishnuvardhan** [comment]
@jasonzhang2022 Regarding item(2), There is `LayerNormalization` [here](https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/keras/layers/LayerNormalization). There is a difference between `LayerNormalization` and `BatchNormalization`.

Regarding item(3), there is a look_ahead and the functionality was described in the tutorial 
```
The look-ahead mask is used to mask the future tokens in a sequence. In other words, the mask indicates which entries should not be used.

This means that to predict the third word, only the first and second word will be used. Similarly to predict the fourth word, only the first, second and the third word will be used and so on.

```
I think this is not an issue. I am closing this, but please let me know if I'm mistaken. Thanks!
